pyvips/tests/test_foreign.py

```diff
@@ -27,6 +34,10 @@ def setUp(self):
         im = pyvips.Image.new_from_file(GIF_FILE)
         self.onebit = im > 128
 
+    @classmethod
+    def tearDownClass(cls):
+        shutil.rmtree(cls.tempdir, ignore_errors=True)
```

Comment:
`ignore_errors` is enabled because some files are not properly removed (they are refusing), see:
```bash
C:\Users\KleisAuke\AppData\Local\Temp\tmpa_g3n91e>dir /s /b /o:gn
C:\Users\KleisAuke\AppData\Local\Temp\tmpa_g3n91e\1wb_s5zp.tif
C:\Users\KleisAuke\AppData\Local\Temp\tmpa_g3n91e\5mle5hjj.tif
C:\Users\KleisAuke\AppData\Local\Temp\tmpa_g3n91e\5vt2sf2s.v
C:\Users\KleisAuke\AppData\Local\Temp\tmpa_g3n91e\65fltyww.tif
C:\Users\KleisAuke\AppData\Local\Temp\tmpa_g3n91e\7avbtfsv.v
C:\Users\KleisAuke\AppData\Local\Temp\tmpa_g3n91e\cpsdel1l.tif
C:\Users\KleisAuke\AppData\Local\Temp\tmpa_g3n91e\g0r0k4bj.hdr
C:\Users\KleisAuke\AppData\Local\Temp\tmpa_g3n91e\gagdxtht.tif
C:\Users\KleisAuke\AppData\Local\Temp\tmpa_g3n91e\h6t3bbvp.tif
C:\Users\KleisAuke\AppData\Local\Temp\tmpa_g3n91e\hjcwqffn.tif
C:\Users\KleisAuke\AppData\Local\Temp\tmpa_g3n91e\ipu5t2ni.tif
C:\Users\KleisAuke\AppData\Local\Temp\tmpa_g3n91e\qz38_gp5.tif
C:\Users\KleisAuke\AppData\Local\Temp\tmpa_g3n91e\r3tfgwou.tif
C:\Users\KleisAuke\AppData\Local\Temp\tmpa_g3n91e\s9uqr4cm.tif
C:\Users\KleisAuke\AppData\Local\Temp\tmpa_g3n91e\sb_n4841.tif
C:\Users\KleisAuke\AppData\Local\Temp\tmpa_g3n91e\z_5wiymf.jpg
```